googleapis/google-cloud-node
# migrate code from googleapis/nodejs-cloud-shell

- feat: initial lib generation
- remove .github files
- feat: add initial template samples
- fix: add gitignore, and samples
- fix: remove additional v1 folder
- feat: add initial samples and tests (#4)
- chore(deps): update dependency sinon to v11 (#3)
- chore: fix package.json repo name (#9)
- chore: release 1.0.0 (#5)
- fix(deps): google-gax v2.17.0 with mTLS (#16)
- chore: release 1.0.1 (#17)
- fix(deps): google-gax v2.17.1 (#19)
- chore: release 1.0.2 (#20)
- chore(nodejs): update client ref docs link in metadata (#26)
- fix(build): migrate to using main branch (#28)
- chore: release 1.0.3 (#29)
- fix(deps): google-gax v2.24.1 (#30)
- chore: release 1.0.4 (#31)
- fix: flag module as GA (#33)
- chore(deps): update dependency mocha to v9 (#36)
- chore: release 1.0.5 (#34)
- build(node): update deps used during postprocessing (#1243) (#44)
- chore: fix the wrong post processor image (#47)
- chore(deps): update dependency @types/node to v16 (#46)
- chore(cloud-rad): delete api-extractor config (#48)
- chore(deps): update dependency sinon to v12 (#49)
- build: add generated samples to .eslintignore (#51)
- docs(node): support "stable"/"preview" release level (#1312) (#54)
- chore: add api_shortname and library_type to repo metadata (#53)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#56)
- test(nodejs): remove 15 add 16 (#1322) (#58)
- chore: Add codeowners (#62)
- chore(deps): update dependency sinon to v13 (#65)
- docs(nodejs): version support policy edits (#1346) (#67)
- feat: upgrade library to ga (#69)
- chore: update repo metadata (#76)
- chore(main): release 1.1.0 (#72)
- chore(deps): update dependency sinon to v14 (#87)
- build!: update library to use Node 12 (#89)
- chore(main): release 2.0.0 (#90)
- chore(deps): update dependency jsdoc-fresh to v2 (#92)
- chore(deps): update dependency jsdoc-region-tag to v2 (#93)
- chore(deps): update dependency linkinator to v4 (#96)
- fix: change import long to require (#99)
- fix: do not import the whole google-gax from proto JS (#1553) (#102)
- chore(main): release 2.0.1 (#100)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
